kpet/run.py

```diff
@@ -18,6 +18,18 @@
 from kpet import data
 
 
+class Case:
+    # pylint: disable=too-few-public-methods,too-many-instance-attributes
+    """A test case in a test suite"""
```

Comment:
This should say "A test case run". As distinct from a "test case", that is `kpet.data.Case`. And also let's not restrict the case to only belonging to a test suite.